openupgrade_scripts/scripts/sale_margin/14.0.1.0/pre-migration.py

```diff
@@ -0,0 +1,26 @@
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, installed_version):
+    openupgrade.logged_query(
+        env.cr,
+        """
+        ALTER TABLE sale_order_line
+        ADD COLUMN margin_percent numeric;
```

Comment:
@hieu1211 sửa lại như này giúp anh nhé, cả bên dưới nữa nhé
```suggestion
        ALTER TABLE sale_order_line
        ADD COLUMN IF NOT EXISTS margin_percent double precision;
```